.github/workflows/shield.yaml

```diff
@@ -0,0 +1,17 @@
+name: "Zephyr Zest_Sensor_IMU CI/CD"
```

Comment:
Shouldn't it be: `Test shield`?

Reply:
Indeed, I forgot to change the PR for shields...